doc/load_data_using_stream_load_howto.md

```diff
@@ -0,0 +1,88 @@
+# Load Data Using Stream Load
+
+The issue is that when the load client residents in a different network other than FE/BE's private network. FE's HTTP
+307 brings BE's private network address to the client who does not recognize and can't process the redirection.
+
+## Issues of Stream Load
```

Comment:
these issues related doc should be part of starrocks documentation, we just need to put a link here and focus on the solution operator provided.

Reply:
done